app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java

```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
             setImageFromPath(path);
         }
 
-        if (!checkColumns())
-        {
-            _fragmentColumns = new FragmentColumns();
+        if (savedInstanceState != null) {
+            _columns = savedInstanceState.getInt(_COUNT_COLUMNS_TAG, 0);
+            _rows = savedInstanceState.getInt(_COUNT_ROWS_TAG, 0);
+            _isSelectedColumns = savedInstanceState
+                    .getInt(_IS_SELECTED_COLUMNS_TAG, 0) != 0;
+            byte[] byteArray = savedInstanceState.getByteArray(_BMP_CURRENT_IMAGE_TAG);
+            if (byteArray != null)
+                _bmpCurrentImage =
+                        BitmapFactory.decodeByteArray(byteArray, 0, byteArray.length);
         }
-        else
-        {
+
+        if (!checkColumns()) {
+            _fragmentColumns = new FragmentColumns();
+        } else {
             _fragmentColumns =
                     (FragmentColumns) getSupportFragmentManager()
-                            .findFragmentByTag(_fragmentColumnsTag);
+                            .findFragmentByTag(_FRAGMENT_COLUMNS_TAG);
         }
         if (getSupportFragmentManager().getBackStackEntryCount() == 0) {
             _fragmentMain = new FragmentMain();
             _fragmentTransaction = getSupportFragmentManager().beginTransaction();
-            _fragmentTransaction.add(R.id.fragment_layout_edit, _fragmentMain, _fragmentMainTag);
+            _fragmentTransaction.add(R.id.fragment_layout_edit, _fragmentMain, _FRAGMENT_MAIN_TAG);
             _fragmentTransaction.addToBackStack(null);
             _fragmentTransaction.commit();
         }
-    }
-
-    /**
-     * Check if columns fragment exists
-     * @return true if existing
-     */
-    public boolean checkColumns()
-    {
-        FragmentColumns myFragment1 = (FragmentColumns) getSupportFragmentManager()
-                .findFragmentByTag(_fragmentColumnsTag);
-        if (myFragment1 != null) {
-            return true;
-        }
-        return false;
+        fragmentConvertDialog = new FragmentConvertDialog();
     }
 
     @Override
-    public void onResume()
-    {
+    public void onResume() {
         super.onResume();
         if (!OpenCVLoader.initDebug()) {
             // Handle error
         } else {
             _baseLoaderCallback.onManagerConnected(LoaderCallbackInterface.SUCCESS);
         }
+        _fragmentMain =
+                (FragmentMain) getSupportFragmentManager()
+                        .findFragmentByTag(_FRAGMENT_MAIN_TAG);
+        if (_fragmentMain != null && _fragmentMain.getView()!= null) {
+            ((ImageView)_fragmentMain
+                    .getView()
```

Comment:
Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md